aws/sagemaker/llm-instance-recommend/lmi_recommender/lmi_config_recommender.py

```diff
@@ -111,8 +126,13 @@ def category_checker(target, category):
     return found
 
 
-def rolling_batch_chooser(model_category: str, instance: dict, dtype: str):
+def strategy_chooser(model_category: str, instance: dict, dtype: str):
```

Comment:
Does the order in this function imply priority?

Reply:
right